### PR TITLE
fixing integration test failure

### PR DESCRIPTION
**Pull request by suryasoma**

### CODE REVIEW

File: spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/QueryPushdownIntegrationTestBase.java [hunk 1]
```diff
@@ -81,21 +82,21 @@ public void testDateFunctionExpressions() {
...
+                    + "TRUNC(report_data_updated_time, 'YEAR') "
                     + "FROM last_updated")
             .collectAsList();
 
     Row r1 = result.get(0);
 
     // Parsing the date rather than setting date to LocalDate.now() because the test will fail
     // in the edge case that the BigQuery read happens on an earlier date
-    LocalDate date = LocalDate.parse(r1.get(0).toString());
+    LocalDate date = LocalDateTime.parse(r1.get(0).toString()).toLocalDate();
```
Comment (VinayLondhe14) [inline spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/QueryPushdownIntegrationTestBase.java:99]: Can probably remove or modify the comment above

**Review by VinayLondhe14 — APPROVED** [review]
